src/clairvoyant/core.cljs

```diff
@@ -62,9 +63,16 @@
 
 (def default-tracer
   (let [pr-val (fn [x]
-                 (if (fn? x)
+                 (cond 
+                   (fn? x) 
                    (pr-str (fn-signature x))
-                   (pr-str x)))
+                   (coll? x)
+                   (pr-str (prewalk (fn [x]
```

Comment:
How about we give the `fn` an optional name of `pr-val` and then simply `(pr-str (prewalk pr-val x))` here?